very_good_core/__brick__/{{project_name.snakeCase()}}/android/app/build.gradle

```diff
@@ -34,12 +34,12 @@ android {
     ndkVersion flutter.ndkVersion
 
     compileOptions {
-        sourceCompatibility JavaVersion.VERSION_17
-        targetCompatibility JavaVersion.VERSION_17
+        sourceCompatibility = JavaVersion.VERSION_1_8
+        targetCompatibility = JavaVersion.VERSION_1_8
     }
 
     kotlinOptions {
-        jvmTarget = 17
+        jvmTarget = JavaVersion.VERSION_1_8
```

Comment:
https://docs.gradle.org/current/javadoc/org/gradle/api/JavaVersion.html
should be JavaVersion.VERSION_18 and not JavaVersion.VERSION_1_8?

Reply:
Hi @dyerjos. Thanks for the comment. We actually are just matching what is part of the basic Flutter `create` template for these values. You can find that here https://github.com/flutter/flutter/blob/a1587bdf2feca871b0482cfb8d9b0a608226b029/packages/flutter_tools/templates/app_shared/android-java.tmpl/app/build.gradle.kts.tmpl#L14

Reply:
Oh good, just double checking. It almost looked like a mistake at first glance